src/AbstractAlgebra.jl

```diff
@@ -919,6 +919,14 @@ include("PrettyPrinting.jl")
 
 include("Deprecations.jl")
 
+################################################################################
+#
+#   Generic functions
+#
+################################################################################
+
+include("GenericFunctions.jl")
```

Comment:
So there is already a file "algorithms/generic_functions.jl" which I think seves the same purpose.

Reply:
Perfect, thank you very much!